### PR TITLE
Revise Polarion Test Case IDs in TALM tests

### PR DESCRIPTION
**Pull request by josclark42**
TALM test cases needed to be revised to remove duplicate test IDs. New polarion test cases were added where needed. Others were merged. Full details of Polarion changes are in each subtask of CNf-12600

### CODE REVIEW

File: tests/cnf/ran/talm/tests/talm-backup.go [hunk 1]
```diff
@@ -150,7 +150,7 @@ var _ = Describe("TALM backup tests", Label(tsparams.LabelBackupTestCases), func
 			Expect(errorList).To(BeEmpty(), "Failed to clean up test resources on spokes")
 		})
 
-		It("should not affect backup on second spoke in same batch", func() {
+		It("should not affect backup on second spoke in same batch", reportxml.ID("74752"), func() {
```
Comment (klaskosk) [inline tests/cnf/ran/talm/tests/talm-backup.go:153]: Needs a comment above for the new test case

File: tests/cnf/ran/talm/tests/talm-batching.go [hunk 2]
```diff
@@ -199,46 +199,47 @@ var _ = Describe("TALM Batching Tests", Label(tsparams.LabelBatchingTestCases),
 				Spoke2APIClient, tsparams.CatalogSourceName, tsparams.TemporaryNamespace).Exists()
 			Expect(catSrcExistsOnSpoke2).To(BeFalse(), "Catalog source exists on spoke 2")
 		})
+		It("should continue the CGU when the first batch fails with the Continue batch timeout action",
+			reportxml.ID("74753"), func() {
+				// 74753 upgrade failure of first batch would not affect second batch
```
Comment (klaskosk) [inline tests/cnf/ran/talm/tests/talm-batching.go:204]:
```suggestion
		})
		
		// 74753 upgrade failure of first batch would not affect second batch
		It("should continue the CGU when the first batch fails with the Continue batch timeout action",
			reportxml.ID("74753"), func() {
```

File: tests/cnf/ran/talm/tests/talm-blockingcr.go [hunk 3]
```diff
@@ -136,9 +136,9 @@ var _ = Describe("TALM Blocking CRs Tests", Label(tsparams.LabelBlockingCRTestCa
 		})
 	})
 
-	When("a blocking CR is missing", func() {
-		// 47948 - Tests multiple UOCRs can be enabled in parallel with blocking CR.
-		It("verifies CGU is blocked until blocking CR created and succeeded", reportxml.ID("47948"), func() {
+	When("a blocking CR is missing", reportxml.ID("47956"), func() {
```
Comment (klaskosk) [inline tests/cnf/ran/talm/tests/talm-blockingcr.go:139]:
```suggestion
	When("a blocking CR is missing", func() {
```

File: tests/cnf/ran/talm/tests/talm-batching.go [hunk 4]
```diff
@@ -199,46 +199,47 @@ var _ = Describe("TALM Batching Tests", Label(tsparams.LabelBatchingTestCases),
 				Spoke2APIClient, tsparams.CatalogSourceName, tsparams.TemporaryNamespace).Exists()
 			Expect(catSrcExistsOnSpoke2).To(BeFalse(), "Catalog source exists on spoke 2")
 		})
+		It("should continue the CGU when the first batch fails with the Continue batch timeout action",
```
Comment (yliu127) [inline tests/cnf/ran/talm/tests/talm-batching.go:202]: @klaskosk is there a  way to format this without additional indentation in the test body? I found it's a bit unfortunate that if our test description exceeds certain number of chars, then the whole test body will be affected.

File: tests/cnf/ran/talm/tests/talm-precache.go [hunk 5]
```diff
@@ -57,7 +57,7 @@ var _ = Describe("TALM precache", Label(tsparams.LabelPreCacheTestCases), func()
 				}
 			})
 
-			It("tests for precache operator with multiple sources", func() {
+			It("tests for precache operator with multiple sources", reportxml.ID("48902"), func() {
```
Comment (klaskosk) [inline tests/cnf/ran/talm/tests/talm-precache.go:60]: Needs comment

File: tests/cnf/ran/talm/tests/talm-precache.go [hunk 6]
```diff
@@ -84,7 +84,7 @@ var _ = Describe("TALM precache", Label(tsparams.LabelPreCacheTestCases), func()
 				Expect(errorList).To(BeEmpty(), "Failed to clean up test resources on hub")
 			})
 
-			It("tests for ocp cache with version", func() {
+			It("tests for ocp cache with version", reportxml.ID("47950"), func() {
```
Comment (klaskosk) [inline tests/cnf/ran/talm/tests/talm-precache.go:87]: Needs comment

File: tests/cnf/ran/talm/tests/talm-precache.go [hunk 7]
```diff
@@ -149,7 +149,7 @@ var _ = Describe("TALM precache", Label(tsparams.LabelPreCacheTestCases), func()
 				Expect(errList).To(BeEmpty(), "Failed to clean up test resources on hub")
 			})
 
-			It("tests for ocp cache with image", func() {
+			It("tests for ocp cache with image", reportxml.ID("48903"), func() {
```
Comment (klaskosk) [inline tests/cnf/ran/talm/tests/talm-precache.go:152]: Needs comment

**Review by klaskosk — CHANGES_REQUESTED** [review]
Thanks for getting these updated, just a few minor changes

File: tests/cnf/ran/talm/tests/talm-batching.go (hunk 4 above)
Comment (klaskosk) [reply tests/cnf/ran/talm/tests/talm-batching.go]: @yliu127 I don't think there's any way to break it into multiple lines without then indent being formatted in.  I do try to edit down the length of the messages if it's just a few characters off

Comment (klaskosk) [reply tests/cnf/ran/talm/tests/talm-batching.go]: Or maybe I'm wrong lol

File: tests/cnf/ran/talm/tests/talm-backup.go (hunk 1 above)
Comment (josclark42) [reply tests/cnf/ran/talm/tests/talm-backup.go:154]: Done.

File: tests/cnf/ran/talm/tests/talm-batching.go (hunk 2 above)
Comment (josclark42) [reply tests/cnf/ran/talm/tests/talm-batching.go]: Done

File: tests/cnf/ran/talm/tests/talm-blockingcr.go (hunk 3 above)
Comment (josclark42) [reply tests/cnf/ran/talm/tests/talm-blockingcr.go]: Done

File: tests/cnf/ran/talm/tests/talm-precache.go (hunk 5 above)
Comment (josclark42) [reply tests/cnf/ran/talm/tests/talm-precache.go:61]: Done

File: tests/cnf/ran/talm/tests/talm-precache.go (hunk 6 above)
Comment (josclark42) [reply tests/cnf/ran/talm/tests/talm-precache.go:89]: Done

File: tests/cnf/ran/talm/tests/talm-precache.go (hunk 7 above)
Comment (josclark42) [reply tests/cnf/ran/talm/tests/talm-precache.go:155]: Done

File: tests/cnf/ran/talm/tests/talm-batching.go [hunk 8]
```diff
@@ -199,8 +199,10 @@ var _ = Describe("TALM Batching Tests", Label(tsparams.LabelBatchingTestCases),
 				Spoke2APIClient, tsparams.CatalogSourceName, tsparams.TemporaryNamespace).Exists()
 			Expect(catSrcExistsOnSpoke2).To(BeFalse(), "Catalog source exists on spoke 2")
 		})
-
-		It("should continue the CGU when the first batch fails with the Continue batch timeout action", func() {
+		// 74753 upgrade failure of first batch would not affect second batch
+		It("should continue the CGU when the first batch fails with the Continue batch timeout"+
+			"action", reportxml.ID("74753"), func() {
```
Comment (yliu127) [inline tests/cnf/ran/talm/tests/talm-batching.go:204]: @klaskosk @josclark42 fyi seems both lint and IDE are happy with this format.

**Review by yliu127 — APPROVED** [review]

File: tests/cnf/ran/talm/tests/talm-batching.go (hunk 8 above)
Comment (klaskosk) [reply tests/cnf/ran/talm/tests/talm-batching.go:204]: I'll have to keep this in mind for other cases so it doesn't indent, thanks

File: tests/cnf/ran/talm/tests/talm-batching.go [hunk 9]
```diff
@@ -199,8 +199,10 @@ var _ = Describe("TALM Batching Tests", Label(tsparams.LabelBatchingTestCases),
...
+			// 74753 upgrade failure of first batch would not affect second batch
```
Comment (klaskosk) [inline tests/cnf/ran/talm/tests/talm-batching.go:205]:
```suggestion
		
		// 74753 upgrade failure of first batch would not affect second batch
		It("should continue the CGU when the first batch fails with the Continue batch timeout"+
			"action", reportxml.ID("74753"), func() {
```

**Review by klaskosk — CHANGES_REQUESTED** [review]
One more minor change but otherwise looks good

File: tests/cnf/ran/talm/tests/talm-blockingcr.go [hunk 10]
```diff
@@ -137,8 +137,8 @@ var _ = Describe("TALM Blocking CRs Tests", Label(tsparams.LabelBlockingCRTestCa
 	})
 
 	When("a blocking CR is missing", func() {
-		// 47948 - Tests multiple UOCRs can be enabled in parallel with blocking CR.
-		It("verifies CGU is blocked until blocking CR created and succeeded", reportxml.ID("47948"), func() {
+		// 47956 - Tests multiple UOCRs can be enabled in parallel with missing blocking CR.
```
Comment (yliu127) [inline tests/cnf/ran/talm/tests/talm-blockingcr.go:140]: @josclark42 you can consider updating polarion test description to match the current design? e.g., UOCR > CGU. can be in separate PR as there are probably other test cases need a better name.

Comment (josclark42) [reply tests/cnf/ran/talm/tests/talm-blockingcr.go]: @yliu127 This is an easy, necessary change so I updated the four occurances of UOCR to CGU. I'll update Polarion as well.

**Review by klaskosk — APPROVED** [review]
